o-fish-ios/Views/ReportFlow/Crew/CrewView.swift

```diff
@@ -102,16 +118,35 @@ struct CrewView: View {
     /// Actions
 
     private func onAppear() {
+        if showingPrefilledAlert {
+
+            initialCaptain = report.captain.copy() as? CrewMemberViewModel ?? CrewMemberViewModel()
```

Comment:
as the copy is type-unsafe, this will be redundant cast

Reply:
fixed